test/elements/depayloader_test.exs

```diff
@@ -18,23 +17,20 @@ defmodule DepayloaderTest do
       "[frameid:1e][timestamp:1] you?"
     ]
 
-    options = %Pipeline.Options{
-      elements: [
-        source: %Source{output: inputs, caps: %Packet{type: :custom_packets}},
-        depayloader: %Depayloader{packets_per_frame: 5},
-        sink: Sink
+    structure = [
+        child(:source, %Source{output: inputs, stream_format: %Packet{type: :custom_packets}}),
+        child(:depayloader, %Depayloader{packets_per_frame: 5}),
+        child(:sink, Sink),
+        get_child(:source) |> get_child(:depayloader) |> get_child(:sink)
```

Comment:
I believe we can do it that way:
```suggestion
    structure = child(:source, %Source{output: inputs, stream_format: %Packet{type: :custom_packets}}) |>
        child(:depayloader, %Depayloader{packets_per_frame: 5}) |>
        child(:sink, Sink)
```
In fact, there is no need to wrap a singe "branch" of a pipeline it into an array.

Reply:
This one also applies to some other similar cases in other tests